controllers/storagepolicyquota/storagepolicyquota_controller_unit_test.go

```diff
@@ -248,6 +265,16 @@ func unitTestsReconcile() {
 				})
 			})
 		})
+
+		When("Unable to get CABundle", func() {
+			BeforeEach(func() {
+				validatingWebhookConfiguration.Webhooks = []admissionv1.ValidatingWebhook{}
+			})
+
+			It("returns an error", func() {
+				Expect(err).To(HaveOccurred())
```

Comment:
can we add another Expect() for substring match -- "unable to get CA bundle from secret" ?